ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
It was my initial idea, but the `border-width` is only used if `border-style` is not `none`. That is why I ended up using the `border` shorthand.